freshmaker/pyxis.py

```diff
@@ -151,9 +152,8 @@ def get_latest_bundles(self, index_images):
         :rtype: list
         """
         # we need 'bundle_path_digest' to find ContainerImage of that bundle
-        include_fields = ['data.channel_name', 'data.version',
-                          'data.related_images', 'data.bundle_path_digest',
-                          'data.bundle_path']
+        include_fields = ['data.channel_name', 'data.version', 'data.related_images',
+                          'data.bundle_path_digest', 'data.bundle_path', 'data.csv_name']
```

Comment:
I realized that `data.csv_name` doesn't exist in schema, maybe you were meant to get `data.csv_display_name`?

Reply:
Sorry @qixiang, I forgot to provide context. This is going to be added to the schema in CWFHEALTH-285.